### PR TITLE
net: Speed up initial connection

### PR DESCRIPTION
**Pull request by akshatsikarwar**
/silent

### CODE REVIEW

**Comment by akshatsikarwar** [comment]
Thank you! Will wait for one roboR run.

**Comment by akshatsikarwar** [comment]
Hit-rate 100%

File: tests/setup [hunk 1]
```diff
@@ -345,14 +345,20 @@ setup_db() {
...
         for node in $CLUSTER; do
-            echo "If we can't talk to any of the nodes the test will timeout"
-            out=$($CDB2SQL_EXE ${CDB2_OPTIONS} --tabs --host $node $DBNAME 'select 1' 2>&1)
-            while  [[ "$out" != "1" ]]; do
-                sleep 2
-                out=$($CDB2SQL_EXE ${CDB2_OPTIONS} --tabs --host $node $DBNAME 'select 1' 2>&1)
-                $CDB2SQL_EXE -v ${CDB2_OPTIONS} --tabs --host $node $DBNAME 'select 1' &> $LOGDIR/${DBNAME}.${node}.conn
+            tbl="dummy_$(cut -d'.' -f1 <<< $node)"
+            while :; do
+                $CDB2SQL_EXE ${CDB2_OPTIONS} --tabs --host $node $DBNAME "CREATE TABLE IF NOT EXISTS ${tbl}(i INTEGER)"
```
Comment (nirbhayc) [inline tests/setup:352]: Forgot to tell, could you also add a line of comment explaining why we are moving from 'select 1' to 'create table' for future reference?

Comment (akshatsikarwar) [reply tests/setup:352]: OK

Comment (akshatsikarwar) [reply tests/setup]: Change of plans - let me try first by tweaking initial connection back-off time. It was 5-10 seconds. Perhaps too much.

**Pull request merged**